synapse/storage/data_stores/main/events.py

```diff
@@ -1508,7 +1509,7 @@ def _purge_history_txn(self, txn, room_id, token_str, delete_local_events):
             [(room_id, event_id) for event_id, in new_backwards_extrems],
         )
 
-        logger.info("[purge] finding redundant state groups")
+        logger.info("[purge] finding state groups referenced by deleted events")
 
         # Get all state groups that are referenced by events that are to be
         # deleted. We then go and check if they are referenced by other events
```

Comment:
it's this comment that I'm complaining about at https://github.com/matrix-org/synapse/pull/6295#discussion_r340828088. We don't do any of this stuff here.

Reply:
D'oh. Don't know why I thought you meant the log line